app/subjects/comment-form.cjsx

```diff
@@ -89,7 +85,7 @@ module.exports = React.createClass
               Add a note about this subject
             </div>
 
-            <div className="tabbed-content-tab #{if @state.tab is 1 then 'active' else ''}" onClick={=> @setState({tab: 1})}>
+            <div className="tabbed-content-tab #{if @state.boards.length < 1 then 'hidden'} #{if @state.tab is 1 then 'active' else ''}" onClick={=> @setState({tab: 1})}>
```

Comment:
While you're working on this component, could you convert the clickable divs here to `<button>`, so that the tabs can be clicked from the keyboard as well as with a mouse. I suspect there's a bunch of bad HTML like this throughout Talk.

Reply:
seconding this request.